docs/cellxgene_census_FAQ.md

```diff
@@ -0,0 +1,109 @@
+- If you want easy access to a subset of data from a dataset or from multiple datasets.
+- If you want to get these data with standardized and harmonized cell and gene metadata.
+- If you want to easily load multi-dataset slices into Scanpy or Seurat.
+- If you want to implement out-of-core (a.k.a online) operations for larger-than-memory processes.
+
+
+For example you could easily get "*all T-cells from Lung with COVID-19*" into an [AnnData](https://anndata.readthedocs.io/en/latest/), [Seurat](https://satijalab.org/seurat/), or into memory-sufficient data chunks via [PyArrow](https://arrow.apache.org/docs/python/index.html) or [R Arrow](https://arrow.apache.org/docs/r/). 
+
+
+**You should not use the Census:**
+
+- If you want access to non-standardized cell metadata and gene metadata available in the original [datasets](https://cellxgene.cziscience.com/datasets).
+- If you want access to the author-contributed normalized expression vaues.
+- If you want access to all data from a single dataset.
+
+## Why does the Census not have a normalized layer or embeddings?
+
+The Census does not have normalized counts or embeddings because:
+
+- The original normalized values and embeddings are not harmonized or integrated across datasets and therefore numerically incompatible.
+- We have not chosen nor implemented a general-purpose normalization or embedding generation method to be used across all Census data.
+
+If you have any suggestions for methods that our team should explore please share them with us via a [feature request in the github repository](https://github.com/chanzuckerberg/cellxgene-census/issues/new?assignees=&labels=user+request&template=feature-request.md&title=).
+
+## How does the Census differentiate from other services?
+
+The Census differentiates from existing single-cell services by providing access to the largest corpus of standardized single-cell data from CZ CELLxGENE Discover via [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA/issues/new/choose). 
+
+Thus, single-cell data from about 50 mi cells across >60 K genes, with 11 standardized cell metadata variables and harmonized GENCODE annotations is at your finger tips to:
+
+- Open and read data at low latency from the cloud.
+- Query and access data using metadata filters.
+- Load and create AnnData objects.
+- Load and create Seurat objects.
+- From Python create PyArrow objects, SciPy sparse matrices, NumPy arrays, and pandas data frames.
+- From R create R Arrow objects, sparse matrices (via the Matrix package), and standard data frames and (dense) matrices.
+
+## Can I query human and mouse data in a single query?
+
+At the moment it is not possible to query both mouse and human data in one query. This is due to the data from these organisms using different [organism-specific gene annotations](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#required-gene-annotations).
```

Comment:
"At the moment" suggests that the census plans to solve this in the future? Is that true?